02-introduction.rst

```diff
@@ -10,7 +10,8 @@ and your new friends (among others) are named "vectors", "arrays", "views" or
 Simple example
 --------------
 
-Let's take a very simple example, random walk.
+Let's take a very simple example, random walk. (You can execute any code below
+from the `code` folder, ideally inside an IPython session or Jupyter notebook.)
```

Comment:
Anything would be helpful that doesn't make you stumble over the first `from tools import timeit`. Also, somewhere you say "This is not a Python beginner guide and you should have an intermediate level in Python" so I'd guess once can assume a decent familiarity with Python and/or Jupyter.